app/src/components/staking/DelegationsOverview.vue

```diff
@@ -55,19 +59,34 @@ export default {
   },
   data: () => ({
     delegations: [],
+    undelegations: [],
+    balances: [],
```

Comment:
@Bitcoinera I think you missed this property. If you don't set it, Vue doesn't know it needs to check for value changes of that property. GQL adds this as well, but in the tests GQL is not applied.

Reply:
OK, thank you :pray: